.github/workflows/build.yml

```diff
@@ -73,7 +73,7 @@ jobs:
                 uses: shivammathur/setup-php@v2
                 with:
                     php-version: "${{ matrix.php }}"
-                    ini-values: "phar.readonly=0,zend.exception_ignore_args=Off"
+                    ini-values: "zend.assertions=1,error_reporting=-1,display_errors=On,phar.readonly=0,zend.exception_ignore_args=Off"
```

Comment:
better way for that is to use `ini-file: development`, to have all the configuration tailored for development environments rather than production ones.

We will still need those specific `ini-values` that we have for our CI setup, but it would avoid having to maintain manually the list of development settings.

Reply:
That's anothér way to do it, yes. Ive updated the commit now for this preferences.